proj_utils/Parser.py

```diff
@@ -60,7 +60,7 @@ def parse_isgoodbehaviour(self):
         return False
 
     def create_text_file(self):
```

Comment:
i dont like the directory name. 
dont shorten names (proj?) 
and the word project is redundant
you can just call it "utils"